ios/FluentUI/Vnext/Divider/MSFDivider.swift

```diff
@@ -48,9 +48,6 @@ import SwiftUI
                                                                             })))
         hostingController.disableSafeAreaInsets()
         view.backgroundColor = UIColor.clear
-        let sizeAnchor = orientation == .horizontal ? view.heightAnchor : view.widthAnchor
-        let size = UIScreen.main.devicePixel + 2 * divider.tokens.padding
-        NSLayoutConstraint.activate([sizeAnchor.constraint(equalToConstant: size)])
```

Comment:
When I was looking with Rafael earlier, it seemed like the ambiguous height warnings in the Divider demo were gone, but they seem to have returned? Not sure what I've done to bring them back, though.

Reply:
Not sure either, but I think we determined that the ambiguous height was not caused by of Divider necessarily, but was result of the confused automatically-heighted table view cell?